applications/CoSimulationApplication/tests/co_simulation_test_factory.py

```diff
@@ -12,6 +12,7 @@
 except ImportError:
     numpy_available = False
 
+have_fsi_dependencies = kratos_utils.CheckIfApplicationsAvailable("FluidDynamicsApplication", "StructuralMechanicsApplication", "MappingApplication", "MeshMovingApplication", "ExternalSolversApplication")
```

Comment:
Shouldn't we check if the installation of of PYTHON only version or the actual kratos even before this ?
That is if it is possible. 

I am approving anyway.

Reply:
So far I have not found a way of checking this
Lets speak about this in person